### ISSUE TITLE
Are there any more radical interface changes coming such as those of 27 days ago?

### ISSUE THREAD

**Issue by ZmeiGorynych** [dbrattli/aioreactive#6]
Dag,

the wave of commits 27 days ago aligned the API more with RxPY and introduced quite a number of breaking API changes (subscribe instead of start, etc).

Could you please tell whether you plan any more major changes like that in the near future, and when do you plan to push this last wave of changes onto PyPI?

Thanks a lot!

**Comment by dbrattli** [comment]
Yes, there might be more changes. This project was/is an experiment (alpha) of new ideas and I'm afraid it would just die out if I don't bring it back into the Rx world again. So there will be some more changes to bring it even closer to Rx and RxPY. The changes will be related to async iteration and we will probably need a `to_async_iterable()` function to convert the async observable to an async iterable first. When/if I'm happy with the result, then RxPY-2.0 will be rewritten to build on the ideas of aioreactive (or RxPY-2.0 will be aioreactive).

**Comment by ZmeiGorynych** [comment]
You'll be happy to hear then that I'm about to use it in a real project for a client :)  (well aware of its alpha status, but it's lightweight enough that I'm confident I can fix any gotchas I run into, if I have to), and I also will be giving a couple of talks in the coming months about using it together with Kafka for (soft) real-time processing, which I think is a perfect complement to async python as it offers a natural way to manage all the data that's not currently in flight.
